### PR TITLE
Feature 225 / UI refactor for project list

### PR DESCRIPTION
**Pull request by lilian-delgado-jalafund**
[AB#225](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/225)

# Overview

Refactor UI Design: 
- Add search bar
- New format for project cards (size defined to support large text description)
- New format for CreationDate

### CODE REVIEW

File: src/components/projects/Projects.jsx [hunk 1]
```diff
@@ -30,58 +30,57 @@ function ProjectList() {
...
+          <div className="search-box-icon">
+            <SearchIcon style={{ color: "#ffffff" }} />
+          </div>
+        </div>
+      </div>
+      <div className="projects-list">
+        {projects.length === 0 ? (
+          <div className="empty-file">
+            <DescriptionRoundedIcon
+              style={{ color: "#e2e2e2", fontSize: "10rem", margin: "30px" }}
```
Comment (jorge-lema-jalafund) [inline src/components/projects/Projects.jsx]: can we change this for classes? check this
{ makeStyles } from '@material-ui/core/styles';
*Note:  for all those components on the updated files

**Review by jorge-lema-jalafund — CHANGES_REQUESTED** [review]

File: src/components/projects/Projects.jsx [hunk 2]
```diff
@@ -30,58 +30,57 @@ function ProjectList() {
...
+            />
+            <Typography
+              style={{
+                color: "#d2d2d2",
+                fontSize: "1.2rem",
+                fontWeight: "bold",
+              }}
+            >
+              Sorry, we couldn't complete your request.
+            </Typography>
```
Comment (freddy-perez-jalafund) [inline src/components/projects/Projects.jsx]: Suggestion change the phrase because it can be confused with a backend error

**Review by jorge-lema-jalafund — APPROVED** [review]
Approved!

**Review by freddy-perez-jalafund — APPROVED** [review]
Approved

**Review by gustavo-rojas-jalafund — APPROVED** [review]

**Pull request merged**